dali/pipeline/operators/decoder/nvjpeg_decoder_cpu.h

```diff
@@ -55,12 +63,19 @@ class nvJPEGDecoderCPUStage : public Operator<CPUBackend> {
                                                     GetFormat(output_image_type_)));
       NVJPEG_CALL(nvjpegDecodeParamsSetAllowCMYK(decode_params_[i], true));
     }
+
+    int nbuffers = spec.GetArgument<int>("cpu_prefetch_queue_depth") * batch_size_;
+    PinnedAllocator::PreallocateBuffers(host_memory_padding, nbuffers);
+    pinned_allocator_.pinned_malloc = &PinnedAllocator::Alloc;
+    pinned_allocator_.pinned_free = &PinnedAllocator::Free;
   }
 
   virtual ~nvJPEGDecoderCPUStage() noexcept(false) {
     NVJPEG_CALL(nvjpegDecoderDestroy(decoder_host_));
     NVJPEG_CALL(nvjpegDecoderDestroy(decoder_hybrid_));
     NVJPEG_CALL(nvjpegDestroy(handle_));
+    int nbuffers = spec_.GetArgument<int>("cpu_prefetch_queue_depth") * batch_size_;
+    PinnedAllocator::FreeBuffers(nbuffers);
```

Comment:
`FreeBuffer` shouldn't need the `nbuffers` argument

Reply:
Right, forgot to remove it
Done